tests/src/JIT/Intrinsics/MathFusedMultiplyAdd.cs

```diff
@@ -0,0 +1,186 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+//
+
+using System;
+using System.Runtime.CompilerServices;
+
+namespace MathFusedMultiplyAddTest
+{
+    class Program
```

Comment:
Given that users can manually do `Fma.MultiplyAddScalar(Vector128.CreateScalarUnsafe(x), Vector128.CreateScalarUnsafe(y), Vector128.CreateScalarUnsafe(z))`, it might be good to have a test explicitly around that.

It might also be good to have a couple tests calling `Fma.MultiplyAddScalar()` with other input types (indirections, local vars, other hwintrinsic loads) to help cover cases that should be rejected by this (and make sure nothing asserts) 😄